node.go

```diff
@@ -20,6 +20,7 @@ import (
 
 const (
 	statAverageExecTime = "avg_exec_time_ns"
+	statErrorCount      = "error_count"
```

Comment:
I like `errors` its consistent with the other nodes that have error counts as well.